UKHomeOffice/nmsw-ui
# Test nmsw 1377 UPT banner

Test NMSW-1377 UPT status banner
Test NMSW-1225 Verify Fal6 is removed when user selects NO to passengers

To test
Run with cypress runner
```
npm run cypress:open:dev -- --env MAIL_API_KEY=${MAIL_API_KEY}
```
and choose submitOtherDeclarations.feature
passenger-status-check.feature

or in cmd line
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/submitOtherDeclarations.feature
```
```
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/passenger-status-check.feature

```
